libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -313,8 +314,8 @@ function(add_libclc_builtin_set)
       INTERNALIZE
       TARGET ${builtins_link_lib_tgt}
       INPUTS $<TARGET_PROPERTY:${builtins_link_lib_tmp_tgt},TARGET_FILE>
-        ${ARG_INTERNAL_LINK_DEPENDENCIES}
-      DEPENDENCIES ${builtins_link_lib_tmp_tgt}
+        $<TARGET_PROPERTY:${ARG_INTERNAL_LINK_DEPENDENCIES},TARGET_FILE>
```

Comment:
I'm not sure this will work correctly if `ARG_INTERNAL_LINK_DEPENDENCIES` is a list. You won't be able to find a property on `foo-target;bar-target`, for example. It might silently do the wrong thing, or it might error, I'm not sure.

Reply:
thanks, changed to reading TARGE_FILE from each target